docs/cvl/statements.md

```diff
@@ -169,7 +169,7 @@ all the pessimistic assertions for the program.
 ## `requireInvariant` statements
 
 
-`requireInvariant` is shorthand for `require` of the expression of the invariant where the invariant parameters have to be substituted with the values/ variables for which the invariant should hold.
+`requireInvariant` is shorthand for `require` of the expression of the invariant where the invariant parameters have to be substituted with the values/ variables for which the invariant should hold. If the config option `--prover_args '-requireInvariantsPreRuleSemantics true'` is used, all `requireInvariant` commands are moved globally to before the rule or invariant code and are not required only at the location where `requireInvariant` is used, for details see {ref}`global_requireInvariant`.
```

Comment:
Nit: no space after `/`

Suggestion:

If the...is used, the invariant is not evaluated where the `requireInvariant` occurs in the rule, but instead it is required before the rule starts in the pre-state and for strong invariants after each unresolved function call that modifies the state.  Only the invariant parameters are evaluated at the place where the `requireInvariant` occurs, for details see {ref}`global_requireInvariant`.